app/assets/stylesheets/dashboard_admin.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the dashboard_admin controller here.
```

Comment:
If you don't need this file, you can delete it

Reply:
ok, on it

Reply:
Done.